### DISCUSSION TITLE
Failed test shows up in results but not on the charts

### DISCUSSION THREAD

**Discussion by PhineasPhreakears** [alexjustesen/speedtest-tracker#1309] [Q&A]
My ISP is doing some upgrades recently and I've had some outages. When I check speedtest-tracker the Results Page shows the failed test attempt but the Dashboard charts skip over the failed test point and it appears that all is well. I noticed this recently when my ISP trouble started and I was on ver 0.17.x (sorry didn't check closely). Updated today and problem persists.
Suggestions? Bug? By design?

**Comment by alexjustesen** [comment]
This is by design, the failed tests will show up in the Results table but not in the charting currently.

**Reply by PhineasPhreakears** [reply]
Thanks for the feedback. Is there a reason for that? Are you going to add this in? I feel like the software used to show this info in the past but I can't say for sure. Just don't lose connection very often. Seems like important info especially given that the public page has no access to the results page.
Anyway, thanks for the project. Great work and I've enjoyed using it.

**Reply by alexjustesen** [reply]
In the past it just showed a gap in reported data. I have on my radar creating a new chart for showing failure/success rates though.

**Reply by PhineasPhreakears** [reply]
That's right. I remember now. That was useful. I really don't see a need for another chart. Just show the failed data points as zeros or gaps in the existing charts. I'd go with zeros so that it is very obvious. That keeps all the important info right there on the first chart. Less is more. That's my 2 cents anyway. Thanks for getting back to me.
